client/components/product-list-item.jsx

```diff
@@ -1,24 +1,38 @@
 import React from 'react';
+import ProductDetails from './product-details';
 
 export default class ProductListItem extends React.Component {
+
   render() {
-    return (
-      this.props.products.map((product, index) =>
-        <div className="col-4" key={index}>
-          <div className="alignment card mt-2">
-            <div className="h-50">
-              <div>
-                <img className="card-img-top" src={product.image}></img>
-              </div>
-              <div className="card-body">
-                <div className="product">{product.name}</div>
-                <div className="price">${product.price / Math.pow(10, 2)}</div>
-                <div className="description">{product.shortDescription}</div>
+    if (this.props.view.name === 'catalog') {
+      return (
+        this.props.products.map((product, index) =>
+          <div className="col-4" key={index}>
+            <div className="alignment card mt-2" onClick={(name, params) => this.props.setView(
+              'details',
+              { productId: product.productId })}>
+              <div className="h-50">
+                <div>
+                  <img className="smallImg card-img-top" src={product.image}></img>
+                </div>
+                <div className="card-body">
+                  <div className="product">{product.name}</div>
+                  <div className="price">${product.price / Math.pow(10, 2)}</div>
+                  <div className="description">{product.shortDescription}</div>
+                </div>
               </div>
             </div>
           </div>
+        )
+      );
+    } else {
+      return (
+        <div>
+          <div>
+            <ProductDetails />
+          </div>
         </div>
-      )
-    );
+      );
+    }
```

Comment:
You can remove this condition since you have a separate component being rendered by `App`.